index.yaml

```diff
@@ -447,3 +447,12 @@ indexes:
   - name: author_id
   - name: published_on
     direction: desc
+
```

Comment:
approving this per seanlip's comment https://github.com/oppia/oppia/pull/18024#discussion_r1165563146